src/main.lua

```diff
@@ -94,6 +95,13 @@ local ActionMap = {
 	AuctionPrices = "Auction-Prices",
 	AllowDelegates = "Allow-Delegates",
 	DisallowDelegates = "Disallow-Delegates",
+	-- PRIMARY NAMES
+	RemovePrimaryNames = "Remove-Primary-Names",
+	CreatePrimaryNameClaim = "Create-Primary-Name-Claim",
+	RevokeClaims = "Revoke-Claims",
```

Comment:
nit: change to `RevokePrimaryClaims` or `RevokePrimaryNameClaims` for consistency and update the tag